lib/Twig/Function.php

```diff
@@ -21,7 +21,7 @@ class Twig_Function
     private $options;
     private $arguments = array();
 
-    public function __construct($name, $callable, array $options = array())
+    public function __construct($name, callable $callable, array $options = array())
```

Comment:
this should be nullable IMO, to preserve the case of functions compiled in a special way through a special node class and not matching an actual PHP callable (see the Symfony form function)